versioned-lambda/orb.yml

```diff
@@ -8,7 +8,7 @@ orbs:
 executors:
   lambci-node12:
     docker:
-      - image: lambci/lambda:build-nodejs12.x
+      - image: lambci/lambda:20200721-nodejs12.x
```

Comment:
Missing `build`:
```
- image: lambci/lambda:20200721-build-nodejs12.x
```
https://hub.docker.com/layers/lambci/lambda/20200721-build-nodejs12.x/images/sha256-c4cb47ad928cb03d149c424af779294f6638bc1ad232ecb9c3d9cdba857fb484?context=explore

Reply:
I just spotted that too :)